src/Compilers/CSharp/Test/Emit2/Semantics/CollectionExpressionTests.cs

```diff
@@ -9822,7 +10155,7 @@ static void Main()
                 """;
             var comp = CreateCompilation(source, targetFramework: TargetFramework.Net80);
             comp.VerifyEmitDiagnostics(
-                // 0.cs(24,31): error CS9188: 'MyCollection<int>' has a CollectionBuilderAttribute but no element type.
+                // (24,31): error CS9188: 'MyCollection<int>' has a CollectionBuilderAttribute but no element type.
```

Comment:
I am confused as to how this baseline changed

Reply:
I'm not sure why there was a "0.cs" filename previously since I thought the filename was only added when there were multiple sources.